explainer.md

```diff
@@ -72,6 +72,113 @@ There are still many open questions, but a few big ones:
+
+> 02.  Do features in your specification expose the minimum amount of information
+>      necessary to implement the intended functionality?
+
+A primary use case of the digital credentials API is for selective disclosure of identity properties such as a cryptographic attestation that the user holds a California driver’s license of an adult. The use of selective disclosure, however, is a decision for the verifier website, wallet and credential issuer based on the use case. There are legitimate scenarios, such as creating or recovering an account on a government website, where uniquely identifiable and potentially non-resettable PII is potentially exposed.
+
+The API is designed to expect the use of response encryption so that this PII is exposed only to the requesting server and not any code running in the web page or browser. It is an [open question](https://github.com/WICG/digital-identities/issues/109) whether this response encryption is something we can reasonably enforce at this layer or not.
+
+The API is also designed to require request transparency to enable user agents and operating systems to appropriately inform users about the level of privacy risk involved in the request. A major area of outstanding work is to outline in the specification our privacy and security guidance to implementers, but we already know that the [Chromium implementation](https://docs.google.com/document/d/1L68tmNXCQXucsCV8eS8CBd_F9FZ6TNwKNOaFkA8RfwI/edit) intends to show users stronger warnings in riskier scenarios such as those that lack the use of selective disclosure.
+
+> 03.  Do the features in your specification expose personal information,
+>      personally-identifiable information (PII), or information derived from
+>      either?
+
+Yes, the API is designed to help facilitate communication of PII from secure wallet applications to verifier web servers. 
+
+> 04.  How do the features in your specification deal with sensitive information?
+
+Today online identity verification (eg. for KYC) is usually done by submitting photos of government identity documents (for example, uploading photos containing the PII via the web’s `<input type=file>` mechanism. Due to the privacy and security limits of this approach, credential issuers and verifiers are working to move to the use of wallet applications which can do selective disclosure of cryptographically attested properties. Today those approaches rely on generic web->app communication paths like custom schemes and server-to-server communication. 
+
+The use of wallets aims to improve on the status quo of uploading images in a number of ways:
+ * Enable the use of selective disclosure (eg. for more privacy-friendly age 18+ verification)
```

Comment:
something about being able to do key binding proof of possession?

```suggestion
 * Enable the use of selective disclosure (eg. for more privacy-friendly age 18+ verification)
 * Enable the wallet to demonstrate legitimate possession of a credential by proving possession of key material bound to that credential
```

Reply:
Agreed, thanks you!

Reply:
I might have gotten lost in the various updates to the PR but don't think anything for this made it in?